src/ui/pages/Overview/Positions/Positions.tsx

```diff
@@ -614,6 +618,17 @@ function PositionList({
               // overscan={5} // the library detects window edge incorrectly, increasing overscan just visually hides the problem
               items={items}
             />
+            {dappIndex !== preparedPositions.dappIds.length - 1 ? (
+              <div
+                style={{
+                  height: 1,
+                  width: '100%',
```

Comment:
unnecessary, and line isn't fullbleed in design

Reply:
✅